commit d5b31d55fa1536a5bd08cf85929ac63a8f723467
Author: Jiří Klimeš <jklimes@redhat.com>
Date:   Wed May 23 10:31:12 2012 +0200

    dbus: remove 'max_replies_per_connection' limit from D-Bus configuration
    
    It is leftover from times when D-Bus default limit was 32. Now, it is 8192, see
    http://cgit.freedesktop.org/dbus/dbus/commit/?id=8d3d8ff55739eebd84d0d53a20a025329feafc3b
    See also https://bugs.freedesktop.org/show_bug.cgi?id=50264

Index: network-manager/src/org.freedesktop.NetworkManager.conf
===================================================================
--- network-manager.orig/src/org.freedesktop.NetworkManager.conf	2012-09-11 19:16:05.307271180 +0200
+++ network-manager/src/org.freedesktop.NetworkManager.conf	2012-09-11 19:16:05.535271173 +0200
@@ -179,7 +179,5 @@
                        send_interface="org.freedesktop.NetworkManager"
                        send_member="wake"/>
         </policy>
-
-        <limit name="max_replies_per_connection">512</limit>
 </busconfig>
 
